docs/reference/elasticsearch/rest-apis/api-conventions.md

```diff
@@ -282,7 +282,21 @@ Most APIs that accept a `<data-stream>`, `<index>`, or `<target>` request path p
 
 In multi-target syntax, you can use a comma-separated list to run a request on multiple resources, such as data streams, indices, or aliases: `test1,test2,test3`. You can also use [glob-like](https://en.wikipedia.org/wiki/Glob_(programming)) wildcard (`*`) expressions to target resources that match a pattern: `test*` or `*test` or `te*t` or `*test*`.
 
-You can exclude targets using the `-` character: `test*,-test3`.
+Targets can be excluded by prefixing with the `-` character. This applies to both concrete names and wildcard patterns.
+For example, `test*,-test3` resolves to all resources that start with `test` except for the resource named `test3`.
+It is possible for exclusion to exclude all resources. For example, `test*,-test*` resolves to an empty set.
+An exclusion affects targets listed _before_ it and has no impact on targets listed _after_ it.
+For example, `test3*,-test3,test*` resolves to all resources that start with `test`, including `test3`, because it is included
+by the last `test*` pattern.
+
+{applies_to}`stack: ga 9.3` A dash-prefixed (`-`) expression is always treated as an exclusion.
+
+::::{note}
+In previous versions, such expression is sometimes _not_ treated as an exclusion due to a bug.
```

Comment:
Can we use `removed 9.3` here?

```
{applies_to}`stack: removed 9.3`
```

I think that'd be nicer than a `::::{note}` which highlights this whole paragraph as if it's important.

Reply:
Thanks. I removed the `note` annotation as suggested fb4b24a75094f0ae8d3a1a0148b68d6a8f8052cf